.github/workflows/gtn-news.yml

```diff
@@ -0,0 +1,162 @@
+---
+# Pull news from the Galaxy Training Network's RSS feed and add them as
+# Galaxy Hub posts.
+name: Galaxy Training Network news
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 * * * *
+
+jobs:
+  collect:
+    name: Collect news from the Galaxy Training Network
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install feedparser.
+        run: pip install feedparser~=6.0
+
+      - name: Parse GTN RSS feed.
+        id: parse
+        shell: python -u {0}
+        run: |
+          import json
+          import os
+          from datetime import datetime, timedelta, timezone
+
+          import feedparser
+
+          feedparser.sanitizer._HTMLSanitizer.acceptable_elements.remove("a")
+          feedparser.sanitizer._HTMLSanitizer.acceptable_elements.remove("img")
+
+          feed = feedparser.parse("https://training.galaxyproject.org/training-material/feed.xml")
+
+          posts = {}
+          oldest = datetime.now().astimezone()
+          for entry in feed.entries:
+              feed_title = f"[GTN news] {entry['title']}"
+              feed_link = entry["link"]
+              feed_blurb = entry["summary"]
+              feed_published = entry["published"].split("T")[0]
+              feed_name = feed_link.split("/")[-1].replace(".html", "")
+              posts[f"{feed_published}-{feed_name}"] = {
+                  "title": feed_title,
+                  "link": feed_link,
+                  "blurb": feed_blurb,
+                  "published": feed_published,
```

Comment:
is there an author field on this end that takes plain text? our feed.xmk exposes it

Reply:
If the feed exposes it then it is worth checking, I'll do it.

Reply:
cheers :) appreciate it!